lib/elastic_apm/opentracing.rb

```diff
@@ -293,7 +308,7 @@ def start_span(
 
       def inject(span_context, format, carrier)
         case format
-        when ::OpenTracing::FORMAT_RACK
+        when ::OpenTracing::FORMAT_RACK, ::OpenTracing::FORMAT_TEXT_MAP
           carrier['elastic-apm-traceparent'] =
             span_context.traceparent.to_header
```

Comment:
We had the situation that span_context was a `OpenTracing::SpanContext.NOOP_INSTANCE` which does not know about `traceparent`. It probably derived from a situation where `start_span` returned a `OpenTracing::Span::NOOP_INSTANCE`.

```suggestion
            span_context.traceparent.to_header if span_context.respond_to?(:traceparent)
```

Reply:
thanks!